CONTRIBUTING.md

```diff
@@ -58,82 +58,73 @@ generate it from the relevant entries in `CHANGES.md` at release time.
 
 ### Compilation
 
+#### Opam/Dune
 The server project uses a standard OCaml development setup based on Opam and
 Dune.
 
-To build it, you'll need an environment with the dependencies stated in
-`coq-lsp.opam`.
+1. Install the dependencies (the complete updated list of dependencies can be found in `coq-lsp.opam`).
 
-`make` will compile the server (the `coq-lsp` binary, to be found in
-`_build/install/default/bin/coq-lsp`).
+    ```sh
+    opam install cmdliner yojson uri dune-build-info menhir ocamlfind zarith sexplib ppx_deriving ppx_sexp_conv ppx_compare ppx_hash ppx_import ppx_deriving_yojson
+    ```
+
+2. Initialize submodules (as of today the `main` branch uses some submodules, which we plan to get rid of soon)
+  
+    ```sh
+    make submodules-init
+    ```
```

Comment:
Not sure if we should point out that the v8.x branches indeed don't need submodules, the opam file in these branches will depend on the right Coq version.

Reply:
Mhhh I'll add a comment just in case